data/migrations/20190606155519_create_recipe_table.js

```diff
@@ -0,0 +1,32 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('create_recipe', function(t){
```

Comment:
Ahh ok, this took me a second, I would've called this something like recipe_ingredients